r2i/tensorrt/loader.cc

```diff
@@ -0,0 +1,112 @@
+/* Copyright (C) 2018-2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+*/
+
+#include "r2i/tensorrt/loader.h"
+
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+#include "r2i/imodel.h"
+#include "r2i/tensorrt/model.h"
+
+static void iRuntimeDeleter (nvinfer1::IRuntime *p) {
+  if (p)
```

Comment:
I recommend making the muscle memory to always use brackets even for one line ifs. This is one of the most common bug sources.